lib/semantics/tools.cc

```diff
@@ -319,8 +319,6 @@ bool IsTeamType(const DerivedTypeSpec *derived) {
   return IsDerivedTypeFromModule(derived, "iso_fortran_env", "team_type");
 }
 
-bool IsCoarray(const Symbol &symbol) { return symbol.Corank() > 0; }
```

Comment:
This is a better implementation of `IsCoarray()` than the one that replaces it below. For example, this one returns true for a use-associated coarray and the other one returns false.

Reply:
Thanks.  I wondered about that.  I'll make that change.